nwp/jobs.py

```diff
@@ -65,39 +65,45 @@ def build_config_on_runtime(model: str, run: str, delay: int = 0) -> dict:
 
 
 @daily_partitioned_config(start_date=dt.datetime(2020, 1, 1))
-def ecmwf_daily_partitioned_config_docker(start: dt.datetime, _end: dt.datetime):
+def ecmwf_uk_daily_partitioned_config(start: dt.datetime, _end: dt.datetime) -> dict:
+    """Create a config dict for the nwp-consumer for uk data from ECMWF."""
     config: NWPConsumerConfig = NWPConsumerConfig(
         date_from=start.strftime("%Y-%m-%d"),
         date_to=start.strftime("%Y-%m-%d"),
         source="ecmwf-mars",
-        env_vars=["ECMWF_API_URL", "ECMWF_API_KEY", "ECMWF_API_EMAIL"],
-        docker_volumes=['/mnt/storage_b/data/ocf/solar_pv_nowcasting/nowcasting_dataset_pipeline/NWP/ECMWF:/tmp'],
-        zarr_dir='/tmp/zarr',
-        raw_dir='/tmp/raw',
+        env_overrides={"ECMWF_AREA": "uk"},
+        zarr_dir='/mnt/storage_b/data/ocf/solar_pv_nowcasting/nowcasting_dataset_pipeline/NWP/ECMWF/uk/zarr',
+        raw_dir='/mnt/storage_b/data/ocf/solar_pv_nowcasting/nowcasting_dataset_pipeline/NWP/ECMWF/uk/raw',
```

Comment:
I made a directory called "UK" (in caps) here where it seems you are looking to save. Not sure how that affects what you're doing but I can move over what i have done merging the ECMWF into there.